website/content/docs/install/index.mdx

```diff
@@ -198,14 +198,32 @@ Nomad uses CNI plugins to configure network namespaces when using the `bridge`
 network mode. All Linux Nomad client nodes using network namespaces must have
 CNI plugins installed.
 
+<h4>Install reference CNI plugins</h4>
+
 The following commands install the CNI reference plugins.
 
 ```shell-session
-$ curl -L -o cni-plugins.tgz "https://github.com/containernetworking/plugins/releases/download/v1.5.0/cni-plugins-linux-$( [ $(uname -m) = aarch64 ] && echo arm64 || echo amd64)"-v1.5.0.tgz && \
+$ export ARCH_CNI=$( [ $(uname -m) = aarch64 ] && echo arm64 || echo amd64)
+$ curl -L -o cni-plugins.tgz "https://github.com/containernetworking/plugins/releases/download/v1.5.0/cni-plugins-linux-${ARCH_CNI}"-v1.5.0.tgz && \
   sudo mkdir -p /opt/cni/bin && \
   sudo tar -C /opt/cni/bin -xzf cni-plugins.tgz
 ```
 
+<h4>Install consul-cni CNI plugin</h4>
+
+If the `transparent_proxy` block is used for Consul Service Mesh, then the 
+`consul-cni` CNI Plugin must be installed on each client node to redirect inbound 
+and outbound traffic for services to the Envoy proxy. The following commands
+assume that reference CNI plugins have already been installed. 
+
+```shell-session
+$ export ARCH_CNI=$( [ $(uname -m) = aarch64 ] && echo arm64 || echo amd64)
+$ curl -L -o consul-cni.zip "https://releases.hashicorp.com/consul-cni/1.5.0/consul-cni_1.5.0_linux_${ARCH_CNI}".zip && \
+  sudo unzip consul-cni.zip -d /opt/cni/bin 
+```
+
+<h4>Configure bridge network to route traffic through iptables</h4>
```

Comment:
This step isn't related to installing the CNI plugins. So we should either change it to this:

```suggestion
<h3>Configure bridge network to route traffic through iptables</h3>
```

Or change the `<h3>Install CNI plugins</h3>` title to something related to overall networking. (Same for the other copy below)

Reply:
Sounds good. Just changed.